src/interop/entityid.rs

```diff
@@ -0,0 +1,92 @@
+#![allow(dead_code)]
+use std::hash::Hash;
+
+use crate::raw::root::RED4ext as red;
+
+#[derive(Debug, Clone)]
+#[repr(transparent)]
+pub struct EntityId(red::ent::EntityID);
+
+impl PartialEq for EntityId {
+    fn eq(&self, other: &Self) -> bool {
+        self.0.hash == other.0.hash
+    }
+}
+
+impl PartialEq<red::ent::EntityID> for EntityId {
+    fn eq(&self, other: &red::ent::EntityID) -> bool {
+        self.0.hash.eq(&other.hash)
+    }
+}
+
+impl PartialEq<EntityId> for red::ent::EntityID {
+    fn eq(&self, other: &EntityId) -> bool {
+        self.hash.eq(&other.0.hash)
+    }
+}
+
+impl Eq for EntityId {}
+
+impl PartialOrd for EntityId {
+    fn partial_cmp(&self, other: &Self) -> Option<std::cmp::Ordering> {
+        self.0.hash.partial_cmp(&other.0.hash)
```

Comment:
the standard convention in Rust is to delegate PartialOrd to Ord
```suggestion
        Some(self.cmp(other))
```

Reply:
done in 7531b56